config_template.toml

```diff
@@ -13,27 +13,21 @@ dt = 0.1
 transfreq = [4.10595, 4.81526]
 # Self-kerr frequencies for each oscillator ("\xi_k", multiplying a_k^d a_k^d a_k a_k, GHz)
 selfkerr = [0.2198, 0.2252]
-# Cross-kerr coupling frequencies for each oscillator coupling k<->l ("\xi_kl", multiplying a_k^d a_k a_l^d a_l, GHz).
-# Format: xi_01, xi_02, xi03, ... ,xi_12, xi_13, ...
-crosskerr = [0.1]
-# Dipole-dipole coupling frequencies for each oscillator coupling k<->l ("J_kl", multiplying a_k^d a_l + a_k a_l^d, GHz).
-# Format Jkl = J_01, J_02, ..., J12, J13, ...
-Jkl = [0.0]
+# Cross-kerr coupling strength for oscillators k<->l ("\xi_kl", multiplying a_k^d a_k a_l^d a_l, GHz).
+# Format: table with keys "k-l" (e.g., "0-1") specifying coupling between oscillators k and l. Unspecified couplings default to zero. E.g. { "0-1" = <val>, "2-4" = <val>}
+crosskerr = { "0-1" = 0.1 }
```

Comment:
I still prefer not to put oscillator indexes into the toml keys. Keys should identify which thing you're configuring, not describe the contents of that thing. And you can't make the key an integer (it must be a string), so by putting the oscillator indexes there you add an extra string parsing step.

How about something like this:
```
  crosskerr = [
    { oscillators = [0, 1], value = 0.1 },
    { oscillators = [1, 2], value = 0.05 }
  ]
```